vmas/simulator/environment/gym.py

```diff
@@ -17,18 +17,25 @@ class GymWrapper(gym.Env):
     def __init__(
         self,
         env: Environment,
+        return_numpy: bool = False,
```

Comment:
```suggestion
        return_numpy: bool = True,
```

I think we can change this to true, I know it is slightly bc-breaking but it might be better aligned with gym, wdyt?